Documentation/VMR-re-bootstrapping.md

```diff
@@ -54,5 +55,7 @@ dependency on a new toolset feature until after that feature has been released.
     corresponding arcade commit/version by looking at the [source-manifest.json](https://github.com/dotnet/dotnet/blob/main/src/source-manifest.json).
     1. Update the arcade SDK version in the [global.json](https://github.com/dotnet/dotnet/blob/main/global.json).
     1. Update the arcade dependency commit and version in the [Version.Details.xml](https://github.com/dotnet/dotnet/blob/main/eng/Version.Details.xml).
+1. Update private source-built SDK and artifacts versions
+    1. Update `PrivateSourceBuiltSdkVersion` and `PrivateSourceBuiltArtifactsVersion` in the [Versions.props](https://github.com/dotnet/dotnet/blob/main/eng/Versions.props).
```

Comment:
All of these links of files that need to be updated should really be pointing to the installer repo since that's where the change needs to be made. This applies to other links in this doc as well.

Reply:
I've followed the existing model, that uses VMR links. @MichaelSimons ?

Reply:
That was a bad decision on my part.

Reply:
OK - fixing this now.

Reply:
Fixed with https://github.com/dotnet/source-build/pull/4284/commits/035e76da2c943f00e5e65988fc78b987bf675197